lib/ansible/modules/network/panos/panos_import.py

```diff
@@ -60,6 +63,12 @@
             - URL of the file that will be imported to device.
         required: false
         default: None
+    validate_ssl:
+        description:
+            - Whether or not certificates should be validated
```

Comment:
Please change to
If C(no), SSL certificates will not be validated. This should only set to no used on personally controlled sites using self-signed certificates.